util/statvar_dcid_generator.py

```diff
@@ -161,57 +172,67 @@ def get_stat_var_dcid(stat_var_dict, ignore_props=None):
     # Helper function to add a property to the dcid list.
     def add_prop_to_list(prop: str, svd: dict, dcid_list: list):
         if prop in svd:
-            token = capitalize_process_word(svd[prop])
+            token = _capitalize_process_word(svd[prop])
             if token is not None:
                 dcid_list.append(token)
-            del svd[prop]
+            svd.pop(prop, None)
 
     dcid_list = list()
     denominator_suffix = ''
     svd = copy.deepcopy(stat_var_dict)
 
     if ignore_props is None:
-        ig_p = _default_ignore_props
+        ig_p = _DEFAULT_IGNORE_PROPS
     else:
         ig_p = copy.deepcopy(ignore_props)
-        ig_p.extend(_default_ignore_props)
+        ig_p.extend(_DEFAULT_IGNORE_PROPS)
 
     for prop in ig_p:
-        if prop in svd:
-            del svd[prop]
-
-    if 'measurementDenominator' in svd:
-        denominator_suffix = 'AsAFractionOf_' + svd['measurementDenominator']
-        del svd['measurementDenominator']
+        svd.pop(prop, None)
 
+    # measurementQualifier is added as a prefix
     add_prop_to_list('measurementQualifier', svd, dcid_list)
 
-    if ('statType' in svd) and (svd['statType'].find("measuredValue") == -1):
-        token = capitalize_process_word(svd['statType'])
-        if token in _stat_type_lookup:
-            dcid_list.append(_stat_type_lookup[token])
-            del svd['statType']
-        else:
-            add_prop_to_list('statType', svd, dcid_list)
-    else:
-        del svd['statType']
+    # Add statType if statType is not measuredValue
+    if ('statType' in svd) and (svd['statType'].find('measuredValue') == -1):
+        svd['statType'] = svd['statType'].replace('Value', '')
+        add_prop_to_list('statType', svd, dcid_list)
+    svd.pop('statType', None)
 
+    # Adding measuredProperty and populationType
     add_prop_to_list('measuredProperty', svd, dcid_list)
     add_prop_to_list('populationType', svd, dcid_list)
 
+    # measurementDenominator is added as a suffix
+    if 'measurementDenominator' in svd:
+        md = svd['measurementDenominator']
+        # Special case: PerCapita is directly appended.
+        if md == 'PerCapita':
+            denominator_suffix = 'PerCapita'
+        # MD that are properties (camelCase) are added as Per(MD)
+        elif md[0].islower():
+            denominator_suffix = 'Per' + _capitalize_process_word(md)
```

Comment:
What is an example, comment?

Reply:
Have added a comment and an example in the tests.